warehouse/query-core/src/main/java/datawave/query/util/FuzzyAttributeComparator.java

```diff
@@ -0,0 +1,91 @@
+package datawave.query.util;
+
+import com.google.common.collect.Sets;
+import datawave.query.attributes.Attribute;
+import datawave.query.attributes.Attributes;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public final class FuzzyAttributeComparator {
+    
+    // this is a utility and should never be instantiated
+    private FuzzyAttributeComparator() {
+        throw new UnsupportedOperationException();
+    }
+    
+    public static boolean singleToSingle(final Attribute existingAttribute, final Attribute newAttribute) {
+        return (existingAttribute.getData().equals(newAttribute.getData())
+                        && existingAttribute.getColumnVisibility().equals(newAttribute.getColumnVisibility()) && existingAttribute.getTimestamp() == newAttribute
+                        .getTimestamp());
+    }
+    
+    public static boolean singleToMultiple(final Attributes multipleAttributes, final Attribute singleAttribute) {
```

Comment:
Method args should follow the method name. Single attribute on the left, Multiple attributes on the right.

Reply:
Done and done